docs/typelibrary/math/math.md

```
@@ -0,0 +1,5 @@
+# math
```

markdownlint-cli2 (0.17.2)

Required heading structure
Expected: # Head; Actual: # math

(MD043, required-headings)